app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -266,6 +266,7 @@ ShortFormApplicationController = (
   $scope.validateHouseholdEligibility = (match, callbackUrl) ->
     $scope.clearHouseholdErrorMessage()
     form = $scope.form.applicationForm
+    # skip the check if we're doing an incomeMatch and the applicant has vouchers
     if match == 'incomeMatch' && $scope.applicant.household_vouchers_subsidies == 'Yes'
```

Comment:
@chrisdolendo to your previous comment, no... I added a clarifying comment here. The logic is "if we're doing an incomeMatch check, *and* the person said Yes to having vouchers, then skip the check". If it was a `||` then it would always skip ahead any time you were trying to validate household income regardless of their Yes/No answer to vouchers.